### PR TITLE
add 202. Happy Number

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 15m
Please check my code.

### CODE REVIEW

File: 202.happy-number.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                return True
+
+            elif n in num_list:
+                return False
+
+            num_list.append(n)
+
+    def calculate_digit_squares(self, num: int) -> int:
+        """
+        calculate the sum of the squares of its digits.
```
Comment (ryuji0123) [inline 202.happy-number.py:27]: Check correct docstring styles

File: 202.happy-number.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        """
+        digit_squares = 0
```
Comment (ryuji0123) [inline 202.happy-number.py:29]: Use generator and return the value directly

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I corrected docstring styles and used generator.  Please check my code.

**Review by ryuji0123 — COMMENTED** [review]
Why two files are included?

**Comment by ryuji0123** [comment]
If you write two solutions, just comment out them except the best one.

**Comment by kobakobashu** [comment]
I deleted the wrong file.  Please check my code.

**Review by tkkawa — APPROVED** [review]
LGTM

**Pull request closed without merging**